### PR TITLE
add-priogrm-area-spare-2280

### LINKED ISSUE #2280: Add Program Area: Spare
### Overview
We need to add program area information to the Spare project so that it will show up when we add the program areas filter.

### Action Items
- [x] Review the project.md file model in the Resources section
- [x] Add the following text to the file indicated in the Resources section
```
program-area:  
    - Social Safety Net
```

### Resources/Instructions
File location to make the change: _projects/spare.md

URL for UAT to check the page has been changed: https://github.com/hackforla/website/blob/gh-pages/_projects/spare.md

[Model for project.md files](https://github.com/hackforla/website/wiki/project.md-file-template)

### PR DESCRIPTION
**Pull request by A-BMT02**
Fixes #2280

What changes did you make and why did you make them ?

Added the following lines to _projects/spare.md as requested
    program-area:  
        - Social Safety Net

### CODE REVIEW

**Comment by answebdev** [comment]
Review ETA: 3/11/2022 (end of day)
Availability: 3 hours

**Review by answebdev — DISMISSED** [review]
@A-BMT02 Thank you for submitting your PR. You did a good job adding the lines to `_projects/spare.md`. However, the text you added should fall inside of the dashes (`---`). Currently, your lines appear in lines 30-31. Notice the dash marks on line 28. Please bring your text inside of these. Also, make sure that the indentation of `- Social Safety Net` lines up with the indentation of the other indented text in the file (e.g., `- Remote` in line 24).

spare

In summary:

- [x] Move your text up inside the `---` marks
- [x] Fix the indentation of `- Social Safety Net`
- [x] Please add your PR to the Project Board

Thank you.

**Comment by A-BMT02** [comment]
Thanks for the feedback! 
I have completed the tasks.

**Comment by SAUMILDHANKAR** [comment]
Review ETA: End of day 3/11/22
Availability: Friday 8-10 PM Pacific Time

**Review by SAUMILDHANKAR — APPROVED** [review]
@A-BMT02 Thanks for making the requested changes. I can see the changes in our projects page. Good job!
- Also, for your next issue would request you to include before and after visuals as well. It would be great if you could add the images in this one as well. 
- Once you have made changes, please request a re-review. This helps your reviewer know that changes have been made.

Thank you.
